### PR TITLE
Implement Graphical Representation of score and similar command

### LINKED ISSUE #22: Add Graphs for `score` and similar commands
https://quickchart.io/documentation/send-charts-with-slack-bot/
we can add a subcommand for plaintext as well

### PR DESCRIPTION
**Pull request by aman-singh7**
fixes #22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,7 +167,7 @@ if you're having issues.
 You can use the `dev_docker-compose.yml` file to spin up containers with Redis services easily.
 Use this env variables for the same.
 ```
-REDISURL=redis://redis:6379
+REDIS_URL=redis://redis:6379
```
Comment (dev-ritik) [inline README.md]: Make this change in a separate PR

File: package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
     "url": "https://github.com/sdsmdg/hubot.git"
   },
   "dependencies": {
-    "follow-redirects": "^1.0.0",
+    "follow-redirects": "1.13.0",
```
Comment (dev-ritik) [inline package.json]: I don't think this is required

File: scripts/util.coffee [hunk 3]
```diff
@@ -1,4 +1,4 @@
-https = require('follow-redirects').https
+https = require('follow-redirects').http
```
Comment (dev-ritik) [inline scripts/util.coffee]: I don't think this is required

File: scripts/graph-batch-score.coffee [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+    parse = (json) ->
+        result = []
+        for line in json.toString().split '\n'
+            result.push line.split(',').map Function.prototype.call,
+            String.prototype.trim
+        if result != ""
+            result
+        else
+            false
```
Comment (dev-ritik) [inline scripts/graph-batch-score.coffee]: If this is the same parse function, don't write it again. Use the existing one

File: scripts/graph-batch-score.coffee [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+#   aman-singh7
+
+util = require('./util')
+
+module.exports = (robot) ->
+
+    scorefield = () ->
+        Field = robot.brain.get("scorefield") or {}
+        robot.brain.set("scorefield",Field)
+        Field
```
Comment (dev-ritik) [inline scripts/graph-batch-score.coffee]: If this is the same scorefield function, don't write it again. Use the existing one

File: scripts/graph-batch-score.coffee [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+        else
+            `relative_year = yy + 1`
+        
+        batch = msg.match[1]
+        year =  relative_year - batch
+        
+        util.info (body) ->
+            result = []
+            result = parse body
+            user_name = []
```
Comment (dev-ritik) [inline scripts/graph-batch-score.coffee]: Use the existing code don't re-write

**Review by dev-ritik — CHANGES_REQUESTED** [review]

**Comment by aman-singh7** [comment]
@dev-ritik committed those changes.

File: scripts/detailed-score.coffee [hunk 7]
```diff
@@ -34,8 +35,4 @@ module.exports = (robot) ->
 			msg.send response
 
 
-
-    			
-
-
```
Comment (dev-ritik) [inline scripts/detailed-score.coffee]: Don't make unnecessary changes

File: scripts/detailed-score.coffee [hunk 8]
```diff
@@ -8,6 +8,7 @@
 # 	Pulkit Karira
 
 module.exports = (robot) ->
+
```
Comment (dev-ritik) [inline scripts/detailed-score.coffee:29]: Don't make unnecessary changes

**Review by dev-ritik — CHANGES_REQUESTED** [review]

**Review by dev-ritik — APPROVED** [review]

**Pull request merged**